src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -1164,6 +1164,29 @@ static smt_termt convert_expr_to_smt(
   return extract_op;
 }
 
+static smt_termt convert_expr_to_smt(
+  const pointer_offset_exprt &pointer_offset,
+  const sub_expression_mapt &converted)
+{
+  const auto type =
+    type_try_dynamic_cast<bitvector_typet>(pointer_offset.type());
+  INVARIANT(type, "Pointer offset should have a bitvector-based type.");
+  const auto pointer_expr = pointer_offset.pointer();
+  const auto converted_expr = converted.at(pointer_expr);
+  const std::size_t pointer_width = type->get_width();
```

Comment:
:pick: This is the width of the result of the operation, not the width of a pointer. Hence it is misnamed.